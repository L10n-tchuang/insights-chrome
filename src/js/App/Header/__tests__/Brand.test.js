import React from 'react';
// import { BrowserRouter as Router } from 'react-router-dom';
import configureMockStore from 'redux-mock-store';
import { shallow, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Brand from '../Brand';

const mockStore = configureMockStore();

describe('Brand component', () => {

    let store;

    beforeEach(() => {
        const initialState = {
            chrome: {
                navHidden: false,
                toggleNav: {}
            }
        };

        store = mockStore(initialState);
    });

    it('should render correctly', () => {
        const wrapper = shallow(
            <Brand store={ store } />
        );
        expect(toJson(wrapper)).toMatchSnapshot();
    });

    // TODO: Simulate button click

});