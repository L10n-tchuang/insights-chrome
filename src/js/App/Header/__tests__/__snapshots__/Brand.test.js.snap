// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Brand component should render correctly 1`] = `
<ContextProvider
  value={null}
>
  <Brand
    navHidden={false}
    store={
      Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      }
    }
    toggleNav={[Function]}
  />
</ContextProvider>
`;
